pkg/alternatives/provider_default.go

```diff
@@ -87,13 +67,13 @@ func (this *provider) FindBinaryFor(ctx context.Context, hostOs sys.Os, hostArch
 		return fail(err)
 	}
 	if du == nil {
-		l.Debug("there does no existing imp alternatives exists at location and the download url resolves to empty")
+		l.Debug("there does no existing alternative at location and the download url resolves to empty")
```

Comment:
No existing alternative location and no download url resolves to empty...
passt das so?